### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by dbrgn**

### CODE REVIEW

**Comment by dbrgn** [comment]
Sorry for not waiting for reviews, I want to get an initial version deployed :slightly_smiling_face:

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,9 +5,12 @@ jobs:
       - image: python:3.6
     steps:
       - checkout
+      - run:
+          name: Clone submodules
+          command: git submodule update --init --recursive
```
Comment (rnestler) [inline .circleci/config.yml:10]: Why did this work before?

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,9 +5,12 @@ jobs:
...
       - run:
           name: Install pip dependencies
-          command: pipenv install --dev
+          command: pip install pipenv && pipenv install --dev
```
Comment (rnestler) [inline .circleci/config.yml:13]: And that? Did the Docker image change?

File: .circleci/config.yml (hunk 1 above)
Comment (dbrgn) [reply .circleci/config.yml:10]: It didn't. I pushed the initial config directly to master.

File: .circleci/config.yml (hunk 2 above)
Comment (dbrgn) [reply .circleci/config.yml:13]: It didn't work previously. I pushed the initial config directly to master.

**Pull request merged**